services/infra-api/serverless.yml

```diff
@@ -23,8 +27,32 @@ provider:
   name: aws
   runtime: nodejs14.x
   region: us-east-1
+  iam:
+    role:
+      path: ${self:custom.iamPath}
+      permissionsBoundary: ${self:custom.iamPermissionsBoundaryPolicy}
+      statements:
+        - Effect: 'Allow'
+          Action:
+            - '*'
+          Resource: !GetAtt FirehoseStreamToNewRelic.Arn
 
 resources:
+  Conditions:
+    CreateNRInfraMonitoring:
```

Comment:
We only want this infra to exist once per environment. It doesn't make sense to have these roles in our PR branches since NR is just reading from the entire `dev` environment anyway.